README.md

```diff
@@ -209,6 +210,17 @@ below shows its use:
 <b>Check Output</b>: {{.Check.Output}}
 ```
 
+## Debugging
+
+It can be helpful to run from the command line to debug issues such as authentication. For this you will need an event. The command below assumes you are running this on a sensu backend.
```

Comment:
Actually running on a sensu-backend using the handler as an asset would also give you no joy. The handler command wouldn't be in the normal executable path, so these instructions are misleading.

And really for debugging purposes I can run this on any remote workstation, as long as I manually download the handler executable appropriate for my workstation OS/Arch.  

Probably want to say something about downloading the appropriate build for the environment where you use sensuctl.